### PR TITLE
certify: migrate to Conan v2

### PR DESCRIPTION
**Pull request by valgur**
Specify library name and version:  **lib/1.0**

---

- [ ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by RubenRBS — DISMISSED** [review]
Thanks a lot @valgur, looks great :)

File: recipes/certify/all/conanfile.py [hunk 1]
```diff
@@ -48,29 +56,38 @@ def lazy_lt_semver(v1, v2):
 
         minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
         if not minimum_version:
-            self.output.warn("{} requires C++17. Your compiler is unknown. Assuming it supports C++17.".format(self.name))
+            self.output.warn(
```
Comment (RubenRBS) [inline recipes/certify/all/conanfile.py:59]:
```suggestion
            self.output.warning(
```

**Review by RubenRBS — COMMENTED** [review]
Minor change to fix v2 :)

**Review by RubenRBS — APPROVED** [review]
Looks nice, thanks a lot!

**Review by czoido — APPROVED** [review]

**Pull request merged**